opensrp-chw/src/main/java/org/smartregister/chw/activity/MalariaProfileActivity.java

```diff
@@ -159,6 +197,28 @@ public void onClick(View view) {
         int id = view.getId();
         if (id == R.id.textview_record_malaria) {
             MalariaFollowUpVisitActivity.startMalariaRegistrationActivity(this, MEMBER_OBJECT.getBaseEntityId());
+        } else if (id == org.smartregister.malaria.R.id.textview_record_anc) {
+            if (view.getTag() == ANC) {
+                AncHomeVisitActivity.startMe(this, MEMBER_OBJECT.getBaseEntityId(), false);
+            } else if (view.getTag() == PNC) {
+                PncHomeVisitActivity.startMe(this, PNCDao.getMember(MEMBER_OBJECT.getBaseEntityId()), false);
+            }
+        } else if (id == org.smartregister.malaria.R.id.textview_record_anc_not_done) {
+            saveVisit(CoreConstants.EventType.ANC_HOME_VISIT_NOT_DONE);
+
+            textViewRecordAncNotDone.setVisibility(View.GONE);
+            textViewRecordAnc.setVisibility(View.GONE);
+            visitStatus.setVisibility(View.VISIBLE);
+        } else if (id == org.smartregister.malaria.R.id.textview_undo) {
+            textViewRecordAnc.setVisibility(View.VISIBLE);
+            textViewRecordAncNotDone.setVisibility(View.VISIBLE);
+            visitStatus.setVisibility(View.GONE);
+        } else if (id == org.smartregister.malaria.R.id.textview_edit) {
+            if (view.getTag() == "EDIT_PNC") {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use equals() to compare strings instead of '==' or '!='](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5426626)